<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QFrame" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>400</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Devanagari</family>
     <pointsize>27</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>知网数据爬取系统</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="search_Botton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>预览</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputLine">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Devanagari</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="to_Getting">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>爬取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="to_dataControl">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>330</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数据管理</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="maxVisibleItems">
    <number>10</number>
   </property>
   <item>
    <property name="text">
     <string>按作者</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按主题</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>to_Getting</sender>
   <signal>clicked()</signal>
   <receiver>Main</receiver>
   <slot>goToGettingPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>to_dataControl</sender>
   <signal>clicked()</signal>
   <receiver>Main</receiver>
   <slot>goToDataPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>561</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goToGettingPage()</slot>
  <slot>goToDataPage()</slot>
 </slots>
</ui>
